### DISCUSSION TITLE
Welcome to GoXLR-TouchPortal-Plugin Discussions!

### DISCUSSION THREAD

**Discussion by oddbear** [oddbear/GoXLR-TouchPortal-Plugin#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ChotaFallen** [comment]
Hey oddbear,

I really would like trying you plugin, but don´t know how. Even your readme confuses me.

Which is the correct file I have to download?
I downloaded "GoXLR-TouchPortal-Plugin-master" extracted it, but can´t find you goxlrworker.exe or entry.tp. in it there are ".cs" files.
You wrote download the tpp file and extract it, but there is no?

Could you please direct me to the correct files.
